src/main/java/mate/academy/springbootwebgreqit/controller/AdminController.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.springbootwebgreqit.controller;
+
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/admin")
+public class AdminController {
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @GetMapping("/dashboard")
+    public String getAdminDashboard() {
```

Comment:
did you see the tutorials from security module?

Pleasem check task description again and provide implementation according to it